src/Paramore.Brighter.Outbox.PostgreSql/PostgreSqlOutboxSync.cs

```diff
@@ -254,10 +238,8 @@ public void MarkDispatched(Guid id, DateTime? dispatchedAt = null, Dictionary<st
 
             try
             {
-                using (var command = InitMarkDispatchedCommand(connection, id, dispatchedAt))
-                {
-                    command.ExecuteNonQuery();
-                }
+                using var command = InitMarkDispatchedCommand(connection, id, dispatchedAt);
```

Comment:
You can't use Using Declarations as we are C# 7.3 due to supporting .NET Standard 2.0 if you look in the file section of GitHub you should be able to see the build errors against the files.

(We support netstandard20 as this gives us framework and core support. Dropping framework support is something we might consider in V10, but the principle of a library being as compatible as important remains important to us)